api/actors.yml

```diff
@@ -0,0 +1,149 @@
+models:
+  Actor:
+    type: object
+    properties:
+      kind:
+        type: string
+        readOnly: true
+        enum: [actor]
+      id: 
+        type: string
+      role:
+        type: string
+        enum: [init,account,miner,storageMarket,paymentBroker]
```

Comment:
Unless there's a reason, should probably match the capitalization and names of the spec. `Init`, `Account`, `StorageMiner`, `StorageMarket`, `PaymentChannel`, `MultisigAccount`.